example_config/capture-acli-auth.py

```
@@ -0,0 +1,289 @@
+        master = dump.DumpMaster(opts, with_termlog=False, with_dumper=False)
+        master.addons.add(recorder)
+        # Signal readiness once the CA cert exists on disk.
+        async def _watch_ready():
+            ca = Path(confdir) / "mitmproxy-ca-cert.pem"
+            for _ in range(200):
+                if ca.exists():
+                    ready_event.set()
+                    return
+                await _asyncio.sleep(0.05)
+            ready_event.set()  # proceed anyway; cert may still appear
+        _asyncio.ensure_future(_watch_ready())
```

Ruff (0.15.21)

[warning] Store a reference to the return value of `asyncio.ensure_future`

(RUF006)